hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/BinaryCompatibilityFileGenerator.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
```

Comment:
Wrong copyright year!

Reply:
Good catch, updated the template [here](https://github.com/hazelcast/hazelcast-client-protocol/pull/115/commits/065562db8cdbe7ec7afd9ff644ec94c98608797d), will re-generate the files and update this PR.